### PR TITLE
i18n(es): translate `components/icons`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Translate `components/icons` to spanish.

### CODE REVIEW

File: docs/src/content/docs/es/components/icons.mdx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+## Importación
+
+```tsx
+import { Icon } from '@astrojs/starlight/components';
+```
+
+## Uso
+
+Muestra un icono usando el componente `<Icon>`.
+Un icono requiere un [`name`](#name) establecido en [uno de los iconos integrados de Starlight](/reference/icons/#todos-los-iconos) y opcionalmente puede incluir un [`label`](#label) para proporcionar contexto a los lectores de pantalla.
```
Comment (dreyfus92) [inline docs/src/content/docs/es/components/icons.mdx:32]:
```suggestion
Un icono requiere un [`name`](#name) establecido en [uno de los iconos integrados de Starlight](/es/reference/icons/#todos-los-iconos) y opcionalmente puede incluir un [`label`](#label) para proporcionar contexto a los lectores de pantalla.
```

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :thumbsup:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a comment regarding a missing translation but everything else looks good to me.

**Comment by dreyfus92** [comment]
Thanks again @HiDeoo 😁

**Review by HiDeoo — APPROVED** [review]
Thanks for translating the remaining labels :raised_hands:

**Pull request merged**